Demos/SwiftUI Demo/SwiftUI Demo/ContentView.swift

```diff
@@ -121,6 +123,19 @@ struct ContentView: View {
                     }
                 }
             }
+
+            Button("Alipay") {
+                alipayPlugin.start(LogtoSocialPluginConfiguration(
+                    redirectUri: URL(string: "alipay://?app_id=some_app_id&state=foo")!,
```

Comment:
How does it work with an app id this arbitrary?

Reply:
replaced before commit :-)

Reply:
Aha.. no wonder